Sources/Datadog/DatadogConfiguration.swift

```diff
@@ -265,6 +265,7 @@ extension Datadog {
         private(set) var firstPartyHosts: Set<String>?
         private(set) var logEventMapper: LogEventMapper?
         private(set) var spanEventMapper: SpanEventMapper?
+        private(set) var loggingSamplingRate: Float?
```

Comment:
We can make this non-optional, to align with other sampling rates in the config.